src/databricks/labs/ucx/source_code/jobs.py

```diff
@@ -139,11 +140,23 @@ def _register_spark_python_task(self, graph: DependencyGraph):  # pylint: disabl
         path = WorkspacePath(self._ws, notebook_path)
         return graph.register_notebook(path)
 
-    def _register_python_wheel_task(self, graph: DependencyGraph):  # pylint: disable=unused-argument
+    def _register_python_wheel_task(self, graph: DependencyGraph) -> Iterable[DependencyProblem]:
```

Comment:
Some code will be moved outside of this method into other classes, but I would like to get feedback:
1. Is this the approach that we prefer?
2. Where should locating the `.dist-info` path go?
3. Where should the using of the `Distribution` go?